### ISSUE TITLE
Site search with a slash / in the query returns 404 or isn't properly encoded

### ISSUE THREAD

**Issue by brendanheywood** [theCrag/website#3020]
**What happened?**

![image](https://user-images.githubusercontent.com/187449/35217961-1af59c80-ffc1-11e7-8ce6-534cb940b1cc.png)

Results in 404

https://www.thecrag.com/nodes/fuzzy-search/a%20%2F%20b?S=a+%2F+b&ST=w

It also doesn't break, but does fail when doing the same search from the facet page:

https://brendan.thecrag.com/nodes/fuzzy-search/nought%20/%20crosses/at/12094657/?sortby=at,desc

**What you expected:**

* [ ] search to work for terms including a slash, some climb names do include a slash (eg quasi alternate names all in one name)
* [ ] facet page search form to also work

**Comment by brendanheywood** [comment]
It is highly likely that by the time this gets to perl land it's indistinguishable from a normal slash in a url and the issue is partially at the apache layer:

https://httpd.apache.org/docs/2.4/mod/core.html#allowencodedslashes

I've run into this before a few times

**Comment by scd** [comment]
Yes I tested this and it seems as though it does not even get to perl land. I think you have found the right reference.

**Comment by brendanheywood** [comment]
It's not the same issue but there is kinda some similarity with this escaping issue in the facets:

https://github.com/theCrag/website/issues/2845

ie a space encoded as a '+' vs %20 or perhaps just a new way of treating the separator

**Comment by brendanheywood** [comment]
This is mostly mitigated because we no longer use the nodes facet as the primary search. but the bug still exists in the node facet.
